src/transformer/cli.js

```diff
@@ -30,22 +30,22 @@
 /* eslint-disable import/default */
 
 import fs from 'fs';
-import path from 'path';
 import yargs from 'yargs';
 import ora from 'ora';
-import createValidator from './validate';
+import path from 'path';
 
-export default async function ({name, transformCallback, validateCallback}) {
-	const validate = createValidator(validateCallback);
+export default async function (transformerSettings) {
```

Comment:
Use object destructuring (And anon function declaration so that we can leverage arrow functions):
```js
export default async ({name, yargsOptions, callback}) => {
```